docs/platform/api/javascript-sdk-options.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: SDK Options Reference
+description: This page provides the available options for StackBlitz SDK projects.
```

Comment:
❓ This page doesn't have an intro so I needed to make it.

Reply:
```suggestion
description: Modify your StackBlitz project with SDK options.
```